spec/workers/mute_worker_spec.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe MuteWorker do
+  subject { described_class.new.perform(account.id, target_account.id) }
+
+  let(:account)              { Fabricate(:account) }
+  let(:target_account)       { Fabricate(:account) }
+
+  describe '#perform' do
+    describe 'home timeline' do
+      before do
+        allow(FeedManager.instance).to receive(:clear_from_home)
+      end
+
+      it "clears target account's statuses" do
+        subject
+
+        expect(FeedManager.instance).to have_received(:clear_from_home).with(account, target_account)
```

Comment:
I've tried to limit the tests to the interfaces called. Elsewhere the tests actually end up testing the FeedManager's implementation details, which does not feel right to me.

https://github.com/mastodon/mastodon/blob/f1300ad284d8c7877a7406e253cf7558596d9837/spec/services/mute_service_spec.rb#L20-L27